Benchmarks/SuperLinq.Benchmarks/LinqTestData.cs

```diff
@@ -0,0 +1,130 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace SuperLinq.Benchmarks;
+
+[SuppressMessage("Design", "CA1052:Static holder types should be Static or NotInheritable")]
```

Comment:
Adding a justification would help external contributors understand why it's important to suppress this particular analysis rule.